support-data-rest/src/test/java/org/uniprot/api/support/data/configure/service/UniParcConfigureServiceTest.java

```diff
@@ -38,7 +38,7 @@ void getSearchItems() {
         UniParcConfigureService service = new UniParcConfigureService();
         List<AdvancedSearchTerm> result = service.getSearchItems(CONTEXT_PATH);
         assertNotNull(result);
-        assertEquals(14, result.size());
+        assertEquals(15, result.size());
```

Comment:
what made this change? I dont see any change in a main (non test) class.

Reply:
It is coming from store module where we keep the configs.